scene/gui/line_edit.cpp

```diff
@@ -1777,7 +1777,7 @@ Size2 LineEdit::get_minimum_size() const {
 
 	if (expand_to_text_length) {
 		// Add a space because some fonts are too exact, and because caret needs a bit more when at the end.
```

Comment:
It's no longer a "space" technically, but the width of the caret, so this might need a slight rewording.
```suggestion
		// Ensure some space for the caret when placed at the end.
```